pkg/webmiddleware/log.go

```diff
@@ -24,7 +25,14 @@ import (
 
 // Log returns a middleware that logs requests.
 // If logger is nil, the logger will be extracted from the context.
-func Log(logger log.Interface) MiddlewareFunc {
+func Log(logger log.Interface, ignorePaths []string) MiddlewareFunc {
+	ignoreRegex := []*regexp.Regexp{}
```

Comment:
`ignoreRegexes := make([]*regexp.Regexp, 0, len(ignorePaths))`

- init slice with known size
- variable holds set so it's plural